src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,84 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+interface INewMovie {
+  onAdd: (movie: Movie) => void;
+}
+
+interface MovieFields extends Movie {
+  description: string;
+}
+
+export const NewMovie = ({ onAdd }: INewMovie) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+  const [fields, setFields] = useState<MovieFields>({
+    description: '',
+    imdbId: '',
+    imdbUrl: '',
+    imgUrl: '',
+    title: '',
+  });
+
+  const handeSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    onAdd({ ...fields });
```

Comment:
don't see any sense for creating copy of fields 
```suggestion
    onAdd(fields);
```